cni-plugins/sriov-passthrough-cni/README.md

```diff
@@ -27,3 +27,37 @@ plugin on all the relevant K8s nodes.
 │   └── sriov-passthrough-cni
 └── README.md
 ```
+
+## Prow job usage:
+
+### In order to use the CNI and allocate one sriov PF:
+1. Add the following annotation to the prow job yaml (or leave the current annotation unchanged).
+
+`k8s.v1.cni.cncf.io/networks: '[{"interface":"net1","name":"sriov-passthrough-cni","namespace":"multus-cni-ns"}`
+
+2. When the parallel sriov jobs mode is enabled, add the following requests/limits:
```

Comment:
Once we configure the prow jobs to use parallel mode, it means they will have a request/limits already,
and then in case one changes the number of times he calls the CNI, he will need to update the requests/limits accordingly.
It will appear here
https://github.com/kubevirt/project-infra/blob/6e55786ef91dce8f8719f63e5fac4cac13041b25/github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml#L264

Reply:
Thanks for explaining. 
Maybe this explanation, or the summery of it, should appear also in the doc.